config/config.go

```diff
@@ -149,38 +149,42 @@ type Config struct {
+	KubernetesTokenFile                                  string                             `yaml:"kubernetes-token-file"`
+	KubernetesHealthcheck                                bool                               `yaml:"kubernetes-healthcheck"`
+	KubernetesHTTPSRedirect                              bool                               `yaml:"kubernetes-https-redirect"`
+	KubernetesHTTPSRedirectCode                          int                                `yaml:"kubernetes-https-redirect-code"`
+	KubernetesDisableCatchAllRoutes                      bool                               `yaml:"kubernetes-disable-catchall-routes"`
+	KubernetesIngressClass                               string                             `yaml:"kubernetes-ingress-class"`
+	KubernetesRouteGroupClass                            string                             `yaml:"kubernetes-routegroup-class"`
+	WhitelistedHealthCheckCIDR                           string                             `yaml:"whitelisted-healthcheck-cidr"`
+	KubernetesPathModeString                             string                             `yaml:"kubernetes-path-mode"`
+	KubernetesPathMode                                   kubernetes.PathMode                `yaml:"-"`
+	KubernetesNamespace                                  string                             `yaml:"kubernetes-namespace"`
+	KubernetesEnableEndpointSlices                       bool                               `yaml:"enable-kubernetes-endpointslices"`
+	KubernetesEnableEastWest                             bool                               `yaml:"enable-kubernetes-east-west"`
+	KubernetesEastWestDomain                             string                             `yaml:"kubernetes-east-west-domain"`
+	KubernetesEastWestRangeDomains                       *listFlag                          `yaml:"kubernetes-east-west-range-domains"`
+	KubernetesEastWestRangePredicatesString              string                             `yaml:"kubernetes-east-west-range-predicates"`
+	KubernetesEastWestRangeAnnotationPredicatesString    multiFlag                          `yaml:"kubernetes-east-west-range-annotation-predicates"`
+	KubernetesEastWestRangeAnnotationFiltersAppendString multiFlag                          `yaml:"kubernetes-east-west-range-annotation-filters-append"`
+	KubernetesAnnotationPredicatesString                 multiFlag                          `yaml:"kubernetes-annotation-predicates"`
+	KubernetesAnnotationFiltersAppendString              multiFlag                          `yaml:"kubernetes-annotation-filters-append"`
+	KubernetesEastWestRangeAnnotationPredicates          []kubernetes.AnnotationPredicates  `yaml:"-"`
+	KubernetesEastWestRangeAnnotationFiltersAppend       []kubernetes.AnnotationFilters     `yaml:"-"`
+	KubernetesAnnotationPredicates                       []kubernetes.AnnotationPredicates  `yaml:"-"`
+	KubernetesAnnotationFiltersAppend                    []kubernetes.AnnotationFilters     `yaml:"-"`
```

Comment:
why the append suffix?

Reply:
Because we may want to implement Prepend in the future.